src/inc/msquic.h

```diff
@@ -18,15 +18,13 @@ Supported Platforms:
 
 --*/
 
-#ifndef _MSQUIC_
-#define _MSQUIC_
+#ifndef _MSQUIC_SRC_INC_MSQUIC_H_
+#define _MSQUIC_SRC_INC_MSQUIC_H_
 
 #ifdef _WIN32
-#pragma once
```

Comment:
Please don't delete the `#pragma once`. Most compilers understand it.

Reply:
ACK, reply later once I finished the automatic tool.

Reply:
Done, see commit 423b482397ece2c3faa42d565cd774f0af487248.